Mathlib/Algebra/Module/Submodule/Map.lean

```diff
@@ -166,6 +166,12 @@ theorem coe_equivMapOfInjective_apply (f : F) (i : Injective f) (p : Submodule R
   rfl
 #align submodule.coe_equiv_map_of_injective_apply Submodule.coe_equivMapOfInjective_apply
 
+@[simp]
+theorem map_equivMapOfInjective_symm_apply (f : F) (i : Injective f) (p : Submodule R M)
```

Comment:
```suggestion
theorem map_equivMapOfInjective_symm_apply {f : F} (i : Injective f) (p : Submodule R M)
```

Reply:
Although this change makes sense, the fact that the two results above: `equivMapOfInjective` and `coe_equivMapOfInjective_apply` have explicit `f` , make me feel that this one should too for consistency. Of course, the other option would be to make `f` implicit in all three results.

Reply:
As you prefer!